### ISSUE TITLE
Asking for permission 

### ISSUE THREAD

**Issue by seiif1** [sduclos/S52#6]
Hello, I am sorry for bothering and I hope you can give me a little of your time.
First of all I have to introduce myself I am a student in the navy academy of Tunisia and I got really interested  in your ‘ OpENCview project ’ and if possible I want to implement it in my end of the year project which I am building using flutter.
I was wondering if you could give me the permission to use your amazing project and if possible and if it does not disturb  you, help me with it .
I hope I am not asking for to much, thank you for your time and have a great day.

**Comment by sduclos** [comment]
Hi,
My code use OpenGL to accelerate graphics, I don't know if flutter ca do that!
My code is GPL, so you will need to recode it if you want another license.
Yes I'll be happy to help if I can.


Have fun :)
Sylvain Duclos.  

    On Sunday, July 4, 2021, 5:25:17 p.m. GMT-4, seiif1 ***@***.***> wrote:  
 
 


Hello, I am sorry for bothering and I hope you can give me a little of your time.
First of all I have to introduce myself I am a student in the navy academy of Tunisia and I got really interested in your ‘ OpENCview project ’ and if possible I want to implement it in my end of the year project which I am building using flutter.
I was wondering if you could give me the permission to use your amazing project and if possible and if it does not disturb you, help me with it .
I hope I am not asking for to much, thank you for your time and have a great day.

**Comment by sduclos** [comment]
I read a bit more about flutter, it's for Android among other. My my code already run on Android.
Sylvain Duclos.

    On Sunday, July 4, 2021, 8:50:58 p.m. GMT-4, s duclos ***@***.***> wrote:  
 
  Hi,
My code use OpenGL to accelerate graphics, I don't know if flutter ca do that!
My code is GPL, so you will need to recode it if you want another license.
Yes I'll be happy to help if I can.


Have fun :)
Sylvain Duclos.  

    On Sunday, July 4, 2021, 5:25:17 p.m. GMT-4, seiif1 ***@***.***> wrote:  
 
 


Hello, I am sorry for bothering and I hope you can give me a little of your time.
First of all I have to introduce myself I am a student in the navy academy of Tunisia and I got really interested in your ‘ OpENCview project ’ and if possible I want to implement it in my end of the year project which I am building using flutter.
I was wondering if you could give me the permission to use your amazing project and if possible and if it does not disturb you, help me with it .
I hope I am not asking for to much, thank you for your time and have a great day.

**Comment by seiif1** [comment]
Yeah flutter can support native platform languages like c and java and you can even use python and ruby if you want.
Thank you for giving me the time but I have to inform you, I am still a beginner and I might ask some obvious questions 😝

**Comment by sduclos** [comment]
Hi
We should use private email instead of via github. 

You can get my email at the bottom of the README ordirectly at ***@***.***

rgds,
Sylvain Duclos.

    On Monday, July 5, 2021, 11:50:41 a.m. GMT-4, seiif1 ***@***.***> wrote:  
 
 


Yeah flutter can support native platform languages like c and java and you can even use python and ruby if you want.
Thank you for giving me the time but I have to inform you, I am still a beginner and I might ask some obvious questions 😝

**Comment by seiif1** [comment]
yeah i tried but i get this everytime i try to send: 

Your message wasn't delivered to sduclos @ users . sf . net   because the address couldn't be found, or is unable to receive mail.

**Comment by sduclos** [comment]
okay 
thanks for the  info. I log in github but still can read your email address. Must be the same for you!

I try again: sylvain_duclos AT yahoo . com 

Sylvain.
